c_binding/libses.h

```diff
@@ -55,6 +57,25 @@ struct _ses_dev_slot_status {
     uint8_t fault_reqstd        : 1;
     uint8_t fault_sensed        : 1;
     uint8_t app_bypass_b        : 1;
+#else
+    uint8_t app_bypass_a        : 1;
+    uint8_t do_not_remove       : 1;
+    uint8_t enc_bypass_a        : 1;
+    uint8_t enc_bypass_b        : 1;
+    uint8_t ready_to_insert     : 1;
+    uint8_t rmv                 : 1;
+    uint8_t ident               : 1;
+    uint8_t report              : 1;
+
+    uint8_t app_bypass_b        : 1;
+    uint8_t fault_sensed        : 1;
+    uint8_t fault_reqstd        : 1;
+    uint8_t dev_off             : 1;
+    uint8_t bypass_a            : 1;
+    uint8_t bypass_b            : 1;
+    uint8_t dev_bypass_a        : 1;
+    uint8_t dev_bypass_b        : 1;
+#endif
```

Comment:
Both are 16 bits. I added new empty line at line 51 to make sure I didn't miss anything.